ToolBox/FxcmDownloader/Program.cs

```diff
@@ -48,10 +49,11 @@ private static void Main(string[] args)
                 // Load settings from command line
                 var tickers = args[0].Split(',');
                 var allResolutions = args[1].ToLower() == "all";
-                var resolution = allResolutions ? Resolution.Second : (Resolution)Enum.Parse(typeof(Resolution), args[1]);
-                var startDate = DateTime.ParseExact(args[2], "yyyyMMdd", CultureInfo.InvariantCulture);
-                var endDate = DateTime.ParseExact(args[3], "yyyyMMdd", CultureInfo.InvariantCulture);
+                var resolution = allResolutions ? Resolution.Tick : (Resolution)Enum.Parse(typeof(Resolution), args[1]);
 
+                var startDate = DateTime.ParseExact(args[2], "yyyyMMdd HH:mm:ss", CultureInfo.InvariantCulture);
+                var endDate = DateTime.ParseExact(args[3], "yyyyMMdd HH:mm:ss", CultureInfo.InvariantCulture);
+                
```

Comment:
Sorry if I wasn't clear in my previous comment about timestamps, I'll try to explain myself in more detail.
The `IDataDownloader` implementations are meant to be reused in other places (for example a desktop GUI app, history requests, etc). The startUtc and endUtc parameters of the `IDataDownloader.Get` method include date and time because we wanted to allow requests for periods smaller than one full day.
The downloader program, on the other hand, should only accept dates on the command line because the `LeanDataWriter` class saves tick data as one file per day (multiple runs of the program for the same date but different times would get files to be overwritten).

So, to close this PR and have it merged, please restore the command line parsing for dates only and add the following line after:
`endDate = endDate.AddDays(1).AddMilliseconds(-1);`